bin.src/imsim.py

```diff
@@ -130,6 +131,8 @@ def main():
         #
         # https://confluence.slac.stanford.edu/pages/viewpage.action?spaceKey=LSSTDESC&title=SSim+2017-03-23
 
+        airmass = 1.0/np.cos(np.radians(OpsimMetaData['altitude']))
```

Comment:
This is only accurate near zenith.  Perhaps use the better approximation in eqn (3) of Krisciunas & Schaefer:
```
airmass = (1 - 0.96 * sin(90 - altitude) ** 2) ** -0.5
```